### ISSUE TITLE
The default unauthenticatedRedirector doesn't work

### ISSUE THREAD

**Issue by brannon** [auth0/angular-jwt#91]
I'm working through some of the quickstart samples, and using the default unauthenticatedRedirector doesn't work.

My config:

```
    // Configuration for angular-jwt
    jwtOptionsProvider.config({
      tokenGetter: function() {
        return localStorage.getItem('id_token');
      },
      whiteListedDomains: ['localhost'],
      loginPath: '/login',
      unauthenticatedRedirectPath: '/login'
    });
```

The default redirector (https://github.com/auth0/angular-jwt/blob/master/src/angularJwt/services/jwtOptions.js#L19):

```
      var options = {
        urlParam: null,
        authHeader: 'Authorization',
        authPrefix: 'Bearer ',
        whiteListedDomains: [],
        tokenGetter: function() {
          return null;
        },
        loginPath: '/',
        unauthenticatedRedirectPath: '/',
        unauthenticatedRedirector: function(location) {
          location.path(this.unauthenticatedRedirectPath);
        }
      }
```

It's not redirecting, because `this` is `window`, not the options object.  The redirector is called here:

```
      function redirectWhenUnauthenticated() {
        $rootScope.$on('unauthenticated', function() {
          var redirector = config.unauthenticatedRedirector;
          var redirectFn;
          if(Array.isArray(redirector)) {
            redirectFn = redirector[redirector.length - 1];
            redirectFn($location);
            unauthenticate();
          } else {
            redirector($location);
          }
        });
      }
```

I will try working around the issue by supplying my own redirector function.

**Comment by brannon** [comment]
Hmm, it looks like the bug is fixed in master (in authManager.js).

**Comment by brannon** [comment]
Alright, looks like it's fixed in 0.1.3 :) Sorry for the noise.
